compute_add_persistent_disk/main.tf

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START compute_create_persistent_disk]
+resource "google_compute_disk" "default" {
+  name = "disk-data"
+  type = "pd-standard"
```

Comment:
Nit: We're referencing a couple of different types `pd-ssd` in the gcloud tab and `pd-balanced` and `pd-standard` in the instructions right before this section. Mind we add a comment that we're using `pd-standard` since it's the default for compute engine?

https://g3doc.corp.google.com/cloud/blockstore/g3doc/architecture/product-type.md?cl=head#external-names-and-types

Reply:
Done